pkg/controller/wildflyserver/wildflyserver_controller.go

```diff
@@ -33,15 +37,21 @@ import (
 var log = logf.Log.WithName("controller_wildflyserver")
 
 const (
-	httpApplicationPort         int32 = 8080
-	httpManagementPort          int32 = 9990
-	standaloneServerDataDirPath       = "/wildfly/standalone/data"
+	httpApplicationPort           int32 = 8080
+	httpManagementPort            int32 = 9990
+	defaultRecoveryPort           int32 = 4712
+	wildflyServerFinalizer              = "finalizer.wildfly.org"
+	wildflyHome                         = "/wildfly"
+	standaloneServerDataDirSuffix       = "/standalone/data"                // data directory where runtime data is saved
+	wftcDataDirName                     = "ejb-xa-recovery"                 // data directory where WFTC stores transaction runtime data
+	labelMarkerOperatedByService        = "wildfly.org/operated-by-service" // label used to remove a pod from receiving load from service during transaction recovery
+	txnRecoveryScanCommand              = "SCAN"                            // Narayana socket command to force recovery
+	reloadRetryCount                    = 10                                // number of retries when waiting for container reload is done (a retry is 1 second)
```

Comment:
where are you spacing out every reload check for 1 second?

Reply:
thanks. this is a wrong comment. it was unintentionally left here from a previous version. I'm going to fix the comment.